services/black/src/ambassador/BlackClient.cs

```diff
@@ -52,17 +33,16 @@ public BlackClient(string connectionString)
         public async Task SetGuestInformationAsync(SetGuestInformation guestInformation)
         {
             // ToDo I don't know if we should start and stop the endpoint in this call or do it outside
+            // maybe make the ambassador disposable?
             var senderEndpoint = await Endpoint.Start(senderConfiguration).ConfigureAwait(false);
-            //var response = senderEndpoint.Request<GuestInformationSet>(guestInformation).ConfigureAwait(false);
-            // ToDo maybe use routing in constructor hence we can omit the endpoint here
             await senderEndpoint.Send(guestInformation).ConfigureAwait(false);
             await senderEndpoint.Stop().ConfigureAwait(false);
         }
 
         public async Task<GuestInformationResponse> GetGuestInformationAsync(Guid orderId)
         {
             var senderEndpoint = await Endpoint.Start(senderConfiguration).ConfigureAwait(false);
-            var response = await senderEndpoint.Request<GuestInformationResponse>(new RequestGuestInformation()).ConfigureAwait(false);
```

Comment:
i dislike the name "RequestGuestInformation" as it sound more like the actual Request you would send rather than the response. GuestInformationRequest would be more appropriate in my opinion.

Reply:
We should discuss that in our meeting today. Last time we agreed on that convention but I don't like it either.

Reply:
rename to GuestInformationRequest as discused